pipelines/measureCrosstalk.yaml

```diff
@@ -0,0 +1,9 @@
+description: ci_cpp CROSSTALK calibration construction
+instrument: lsst.obs.lsst.Latiss
+imports:
+  - location: $CP_PIPE_DIR/pipelines/Latiss/measureCrosstalk.yaml
+tasks:
+  crosstalkSolve:
+    class: lsst.cp.pipe.measureCrosstalk.CrosstalkSolveTask
+    config:
+      doFiltering: true
```

Comment:
Could you add a 1-line comment on why this needs to be turned on just here?

Reply:
Added.  The input images don't fully test all source-target pairs, so the filtering is needed to exclude pairs that have zero pixels that result in NaN solutions.